spec/system/jobseekers_can_complete_a_job_application_spec.rb

```diff
@@ -2,22 +2,26 @@
 
 RSpec.describe "Jobseekers can complete a job application" do
   let(:jobseeker) { create(:jobseeker) }
+  let(:school) { create(:school) }
+
   let(:vacancy) { create(:vacancy) }
-  let(:job_application) { create(:job_application, jobseeker: jobseeker, vacancy: vacancy) }
 
   before do
     allow(JobseekerAccountsFeature).to receive(:enabled?).and_return(true)
     allow(JobseekerApplicationsFeature).to receive(:enabled?).and_return(true)
     login_as(jobseeker, scope: :jobseeker)
+    vacancy.organisation_vacancies.create(organisation: school)
   end
 
   it "allows jobseekers to complete application and go to review page" do
-    visit jobseekers_job_application_build_path(job_application, :personal_details)
+    visit job_path(vacancy)
```

Comment:
I think we can leave this spec as it is, and instead change `jobseekers_can_start_a_job_application_spec.rb`

Or, we could merge all 3 of the job application system specs in to one larger one?

Reply:
Wherever you do the tests @cesidio, can you include the feature flag in the test?